nginx/templates/multi-channel.conf.tmpl

```diff
@@ -28,13 +28,16 @@
         proxy_ignore_headers Cache-Control;
         proxy_cache_valid 200 302 {{ getenv "CACHE_DURATION_NGINX_OK" }};
         proxy_cache_valid 404     {{ getenv "CACHE_DURATION_NGINX_NF" }};
-        # set $c_uri as cache_key
-        proxy_cache_key $c_uri;
 
         add_header X-icm-channel {{ $channel }} always;
         add_header X-icm-default-lang {{ $lang }} always;
         add_header X-ua-device $ua_device always;
 
+        if ($render_ssr = 0) {
```

Comment:
Same here, I'm not sure what the consequences of putting the if block into the location template would be. The SSR rendering doesn't have to be configurable per baseHref, does it? So maybe we can move it outside.

Reply:
It looks like it doesn't work outside of this location block. Does it work for you?